src/debuginfo.cpp

```diff
@@ -194,7 +194,9 @@ class JuliaJITEventListener: public JITEventListener
     virtual void NotifyFunctionEmitted(const Function &F, void *Code,
                                        size_t Size, const EmittedFunctionDetails &Details)
     {
+        int8_t gc_state = jl_gc_state_save_and_set(3);
```

Comment:
Done. Also for the `1`.

I also change it to `2` instead since I don't think we need it to be a bit flag (which was why I use `3` instead of `2` to begin with).